misc/flatpak/README.asciidoc

```diff
@@ -0,0 +1,65 @@
+Flatpak
+=======
+
+The flatpak is based on the org.kde.Platform version 5.9 and ships all
+additional dependecies of qutebrowser. The platform includes QT 5.9.
+
+To install the SDK system wide run:
+
+ sudo flatpak install org.kde.Sdk/x86_64/5.9
```

Comment:
So I finally wanted to test this, but I get an error on Flatpak 0.10.3 on Archlinux:

```
error: REMOTE and REF must be specified
```

Shouldn't this be `sudo flatpak install flathub org.kde.Sdk/x86_64/5.9` instead?

Reply:
Indeed:
sudo flatpak install flathub org.kde.Sdk/5.9

Reply:
Fixed.

Reply:
sudo flatpak install flathub org.kde.Sdk//5.9